### ISSUE TITLE
Converting one model to another using ONNX

### ISSUE THREAD

**Issue by hitman186** [onnx/onnx#2223]
Can I convert a pytorch model into onnx and then that onnx model to keras and then onnx and finally tensorflow? So that I can validate that ONNX works across all models.

PyTorch -> ONNX -> Keras -> ONNX -> Tensorflow

**Comment by prasanthpul** [comment]
What's your end goal? PyTorch, Keras, and TensorFlow models can be converted to ONNX and inferencing using ONNX Runtime or other engine.

**Comment by hitman186** [comment]
> What's your end goal? PyTorch, Keras, and TensorFlow models can be converted to ONNX and inferencing using ONNX Runtime or other engine.

I want to see whether one model can be converted to all the models and the inference in all those models must be same for me to decide ONNX is working fine across all platforms

**Comment by askhade** [comment]
The right way to do that would be to do tensorflow -> onnx conversion and check accuracy and similarly keras -> onnx and do accuracy check , Pytorch -> onnx and do accuracy check...
